src/lib.rs

```diff
@@ -82,6 +90,11 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 #![doc(html_root_url = "https://docs.rs/crate/apint/0.2.0")]
+// TODO remove these as soon as they are on stable
+#![feature(const_if_match)]
+#![feature(const_fn)]
+#![feature(const_panic)]
+#![feature(const_nonzero_int_methods)]
```

Comment:
We should not move to nightly again. So whatever requires nightly features should either be removed from the crate or be rewritten in a non-breaking manner to no longer require nightly features. This crate should stay on stable Rust definitely.

Reply:
I forgot to remove these, I had added them to check if making the `bw` function const was possible